src/tasks/lm_parsing.py

```diff
@@ -0,0 +1,150 @@
+"""
+Task definitions for language modeling parsing tasks. Used for unsupervised parsing encoders.
```

Comment:
'language modeling parsing' is okay for variable names, but it doesn't make sense in a written out description. Say that these are language modeling tasks, but these particular versions are set up for use with unsupervised parsing/latent-tree models.

Reply:
Updated